build/android/.gitignore

```diff
@@ -0,0 +1,10 @@
+build
+local.properties
+.gradle
+app/build
+app/release
+app/src/main/assets
+native/.*
+native/build
```

Comment:
I miss here
## A
```
.idea/*
!.idea/codeStyles/
```
But it makes no sense, when you don't remove `.idea` from the top most `.gitignore`

## B
`*.iml`